src/signMessage.c

```diff
@@ -105,16 +106,27 @@ void handleSignMessage(
     volatile unsigned int *flags,
     volatile unsigned int *tx
 ) {
-    int data_has_length_prefix = (dataLength & DATA_HAS_LENGTH_PREFIX);
+    int deprecated_host = ((dataLength & DATA_HAS_LENGTH_PREFIX) != 0);
 
-    if (data_has_length_prefix) {
+    if (deprecated_host) {
         dataLength &= ~DATA_HAS_LENGTH_PREFIX;
     }
 
     if ((p2 & P2_EXTEND) == 0) {
         MEMCLEAR(G_derivationPath);
         MEMCLEAR(G_message);
 	    G_messageLength = 0;
```

Comment:
This indentation looks weird...

Reply:
Yes it does... :thinking: 

Good eye!

Reply:
Ah!  There's a tab in here!  No wonder it looks fine locally